channels/urbdrc/client/libusb/request_queue.c

```diff
@@ -71,50 +74,63 @@ TRANSFER_REQUEST* request_queue_register_request(REQUEST_QUEUE* queue, UINT32 Re
 		queue->tail->next = (void*) request;
 		request->prev = (void*) queue->tail;
 		queue->tail = request;
+		request = NULL;
```

Comment:
This results in incorrectly returning NULL even though the item was successfully added to the queue